osgp/platform/osgp-adapter-ws-publiclighting/src/main/java/org/opensmartgridplatform/adapter/ws/publiclighting/infra/jms/messageprocessor/DomainResponseMessageProcessor.java

```diff
@@ -125,10 +125,11 @@ protected void handleMessage(final CorrelationIds ids, final String messageType,
 
         final short numberOfNotificationsSent = 0;
         Serializable deviceResponseObject;
-        if (dataObject == null) {
-            deviceResponseObject = resultDescription;
-        } else {
+
+        if (ResponseMessageResultType.OK.equals(resultType)) {
```

Comment:
ResponseMessageResultType.OK == resultType is also allowed and is easier to read (in my opinion).
The same applies to the DomainResponseMessageProcessor for tariff switching.

Reply:
Done